data/xml/2021.codicrac.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
`codicrac` should be not be recorded as a new venue, but as a volume within `codi` or `crac`, and then cross-listed within the other. I suggest you use `2021.codi-sharedtask` as the volume name, and then add that volume under `crac` in `data/venues/joint.yaml`.

Reply:
Done.